Gems/AWSCore/Code/CMakeLists.txt

```diff
@@ -111,6 +111,9 @@ if (PAL_TRAIT_BUILD_HOST_TOOLS)
         BUILD_DEPENDENCIES
             PRIVATE
                 Gem::AWSCore.Editor.Static
+        RUNTIME_DEPENDENCIES
+            3rdParty::pyside2
```

Comment:
Question: will pyside2 runtime dependencies get copied to AWSCoreEditorQtBin folder? as the tool expects to load runtime dependencies there, see the reason https://jira.agscollab.com/browse/LYN-2669

Reply:
Yes it does